Core/HLE/sceGe.cpp

```diff
@@ -193,13 +193,12 @@ static void __GeExecuteInterrupt(u64 userdata, int cyclesLate) {
 	__TriggerInterrupt(PSP_INTR_IMMEDIATE, PSP_GE_INTR, PSP_INTR_SUB_NONE);
 }
 
+// Should we still do this?
```

Comment:
No - even if we wanted to try to sync CPU cycles with GPU cycles, I think we'd need to do it very differently.  I think `__GeCheckCycles` can be removed entirely.

-[Unknown]